plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/debug/FileResourceLocationHandler.java

```diff
@@ -31,8 +31,8 @@
 @Singleton
 public class FileResourceLocationHandler implements DebuggerLocationHandler {
 
-  private final EditorAgent editorAgent;
-  private final AppContext appContext;
+  protected final EditorAgent editorAgent;
```

Comment:
I don't see a reason to make these fields protected.

Reply:
To be visible in children classes.

Reply:
Yes, but why is that necessary?

Reply:
It is used there.